python_modules/dagstermill/dagstermill/__init__.py

```diff
@@ -225,6 +269,46 @@ def is_json_serializable(value):
         return False
 
 
+class SerializableRuntimeType(Enum):
+    SCALAR = 'scalar'
+    ANY = 'any'
+    PICKLE_SERIALIZATION_STRATEGY = 'pickle'
+    NONE = ''
+
+
+def runtime_type_to_enum(runtime_type):
+    if runtime_type.is_scalar:
+        return SerializableRuntimeType.SCALAR
+    elif runtime_type.is_any:
+        return SerializableRuntimeType.ANY
+    elif runtime_type.serialization_strategy and isinstance(
+        runtime_type.serialization_strategy, PickleSerializationStrategy
+    ):
+        return SerializableRuntimeType.PICKLE_SERIALIZATION_STRATEGY
+
+    return SerializableRuntimeType.NONE
+
+
+def input_name_serialization_enum(runtime_type, value):
+    runtime_type_enum = runtime_type_to_enum(runtime_type)
+
+    if runtime_type_enum == SerializableRuntimeType.ANY:
```

Comment:
`ANY` -> json-serializable?

Reply:
I think we still need the ANY category for outputs, since we don't know if the output value is json_serializable until we have the value (so it might be of type ANY and not json serializable). But added a json_serializable to the enum!